tests/functional/UseCase/Table/Import/FromTable/ImportTableFromTableTest.php

```diff
@@ -561,7 +561,12 @@ public function testPartitionedTableWithRequirePartitionFilter(): void
         } catch (Throwable $e) {
             // 'Cannot query over table 'xxx.xxx' without a filter over column(s) 'id'
             // that can be used for partition elimination'
-            $this->assertInstanceOf(BadExportFilterParametersException::class, $e);
+            $this->assertInstanceOf(BadExportFilterParametersException::class, $e, sprintf(
+                'Expected error instance "%s" not thrown. Got "%s" with message "%s"',
+                BadExportFilterParametersException::class,
+                $e::class,
+                $e->getMessage(),
+            ));
```

Comment:
assert je upravený nevěděl sem proč mě padá test a padal protože sem potřeboval rozšířit try/catch kde to háže exception